google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -244,6 +244,17 @@ public String getStreamName() {
     }
   }
 
+  public static class AppendSerializationError extends AppendSerializtionError {
```

Comment:
Javadoc? (I expect a copy of AppendSerializtionError) Explanation of typo and backward compatibility is nice.

Reply:
Since we don't want to break customers using the AppendSerializationError(with typo) and at the same time, move all the reference we have to that class in the library, to the class without typo, it made sense to use `class AppendSerializationError extends AppendSerializtionError` (super class has typo).

The other way round would mean, we move the code in `AppendSerializtionError` to `AppendSerializationError` for us to use in the library. That would break the external customers, if we don't move the code, that will cause code duplication.